src/screens/settings/SettingScreen.tsx

```diff
@@ -1,113 +1,153 @@
-            }}
-          >
-            <Text>Service Agreement</Text>
-            <Icon
-              name="ios-chevron-forward"
-              color={COLOR.black._800}
-              size={20}
-            />
-          </TouchableOpacity>
-          <TouchableOpacity
-            style={styles.item}
-            onPress={(): void => {
-              console.log('Privacy')
-            }}
-          >
-            <Text>Privacy</Text>
-            <Icon
-              name="ios-chevron-forward"
-              color={COLOR.black._800}
-              size={20}
-            />
-          </TouchableOpacity> */}
-          <View style={styles.item}>
-            <Text>Version</Text>
-            <Text style={{ color: COLOR.primary._400 }}>
-              {version.codepush
-                ? `${version.app}-${version.codepush}`
-                : `${version.app}`}
-            </Text>
-          </View>
-          {/* <TouchableOpacity
-            style={styles.item}
+          />
+        </View>
+        <View style={styles.itemGroup}>
+          <SettingItem
+            name={t('Settings.ChangePin')}
+            onPress={(): void => {}}
+          />
+          <SettingItem
+            name={t('Settings.ExportWallet')}
             onPress={(): void => {
-              console.log('Version')
+              navigation.navigate(Routes.ExportPrivate)
             }}
-          >
-            <Text>Contact</Text>
-            <Icon
-              name="ios-chevron-forward"
-              color={COLOR.black._800}
-              size={20}
-            />
-          </TouchableOpacity> */}
+          />
         </View>
         <View style={styles.itemGroup}>
+          <SettingItem
+            name={t('Settings.ExportWallet')}
+            onPress={(): void => {}}
+          />
+          <SettingItem name={t('Settings.Privacy')} onPress={(): void => {}} />
+          <SettingItem name={t('Settings.Contact')} onPress={(): void => {}} />
+          <SettingItem
+            name={t('Settings.GiveUsFeedback')}
+            onPress={(): void => {}}
+          />
+          <SettingTextItem
+            name={t('Settings.Version')}
+            text={
+              version.codepush
+                ? `${version.app}-${version.codepush} ${
+                    __DEV__ ? '(dev)' : '(release)'
+                  }`
+                : `${version.app} ${__DEV__ ? '(dev)' : '(release)'}`
+            }
+          />
+        </View>
+        <View style={(styles.itemGroup, { alignSelf: 'center' })}>
           <TouchableOpacity
-            style={[styles.item, { justifyContent: 'center' }]}
-            onPress={logout}
+            style={[
+              styles.item,
+              {
+                borderColor: COLOR.black._100,
+                borderWidth: 1,
+                borderRadius: 9,
+                paddingHorizontal: 12,
+                paddingVertical: 6,
+                alignSelf: 'flex-start',
+              },
```

Comment:
add it to styles.item?

Reply:
The design has not been finalized. However, I plan to reorganize styles for all components and screens in the future. This screen will change soon.  (temporary)